tests/test_onco_print.py

```diff
@@ -1,7 +1,10 @@
 import os
+
+from pytest_dash.wait_for import wait_for_element_by_css_selector
```

Comment:
```suggestion
from pytest_dash.wait_for import wait_for_element_by_css_selector, wait_for_elements_by_css_selector
```

Reply:
Aww I'll stack these imports over multiple lines, because it's way too confusing...